python/lsst/pipe/tasks/photoCal.py

```diff
@@ -386,8 +374,6 @@ def extractMagArrays(self, matches, filterName, sourceKeys):
 
         srcMagErrArr = np.array([abMagErrFromFluxErr(sfe, sf)
                                 for sfe, sf in zip(srcInstFluxErrArr, srcInstFluxArr)])
```

Comment:
This should also be vectorized as above.  There are probably others in the code that weren't put into the GitHub change-log, which maybe outside the scope of this ticket, but shouldn't be too hard to do while doing the rearranging.